src/hearings/components/hearing-parties-title/hearing-parties-title.component.ts

```diff
@@ -1,28 +1,34 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
-import { select, Store } from '@ngrx/store';
-import { Subscription } from 'rxjs';
+import { Component, OnInit } from '@angular/core';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { Store, select } from '@ngrx/store';
+import { Subscription, combineLatest } from 'rxjs';
+import { AppConstants } from '../../../app/app.constants';
 import { ServiceHearingValuesModel } from '../../models/serviceHearingValues.model';
 import * as fromHearingStore from '../../store';
 
 @Component({
   selector: 'exui-hearing-parties-title',
   templateUrl: './hearing-parties-title.component.html'
 })
-export class HearingPartiesTitleComponent implements OnInit, OnDestroy {
+export class HearingPartiesTitleComponent implements OnInit {
```

Comment:
Why has OnDestroy been removed here yet is still present?

Reply:
@connorpgpmcelroy - I added it back as it is required. Thank you for finding this one.